### ISSUE TITLE
Include ngx_pagespeed

### ISSUE THREAD

**Issue by CAFxX** [cloudfoundry/staticfile-buildpack#79]
What version of Cloud Foundry are you using?
220

What version of the buildpack you are using?
1.3.9

If you were attempting to accomplish a task, what was it you were attempting to do?
Enable pagespeed via custom nginx.conf

What did you expect to happen?
The pagespeed module to start processing requests

What was the actual behavior?
ngx_pagespeed is not included in the nginx package, so nginx fails because of unknown configuration directive.

We briefly discussed the purpose of pagespeed over https://cloudfoundry.slack.com/archives/buildpacks/p1468242176000666 and decided to file a ticket.

**Comment by CAFxX** [comment]
Starting with ngx_pagespeed 1.10.33 the module can also be built as a dynamic module, i.e. a module that needs to be explicitly loaded in nginx.conf. This means that buildpack users that don't need it have absolutely no additional attack surface or runtime impact.

**Comment by sesmith177** [comment]
We've created a [story](https://www.pivotaltracker.com/story/show/126402469) to further research including the ngx_pagespeed module

**Comment by sesmith177** [comment]
@CAFxX we are going to hold off on adding ngx_pagespeed  for now. Specifically, we're waiting on the arrival of a more complete version of nginx's dynamic module functionality so that users can choose which modules to load

**Comment by CAFxX** [comment]
@sessmith can you elaborate a bit more? AFAICT what you're describing ("users can choose which modules to load") is exactly what dynamic modules are supposed to provide. Unless you mean you want to give users the ability to provide their own modules, but that's definitely out of scope for this ticket...

**Comment by RochesterinNYC** [comment]
@CAFxX what @sesmith177's comment means was that we're looking to potentially add and support Nginx's dynamic modules functionality in the future for this buildpack but it's currently relatively new and we'd like to adopt/support it when it's at a more stable, mature phase. Once dynamic modules are supported by this buildpack, `ngx_pagespeed` can be easily brought in by the app developer/buildpack user.
